app/routers/user.py

```diff
@@ -59,3 +60,29 @@ def get_all_user_events(session: Session, user_id: int) -> List[Event]:
         session.query(Event).join(UserEvent)
         .filter(UserEvent.user_id == user_id).all()
     )
+
+
+def disable_user(session: Session, user_id: int):
+    """this functions changes user status to disabled.
+    returns:
+    True if function worked properly
+    False if it didn't."""
+
+    events_user_owns = list(session.query(Event).filter_by(owner_id=user_id))
+    """if user owns any event, he won't be able to disable himself."""
+    if events_user_owns:
+        return False
+
+    """if user doesn't own any event,
+    we will disable him and remove the user from all of its future events."""
+    user_disabled = session.query(User).get(user_id)
+    user_disabled.disabled = True
+    future_events_for_user = session.query(UserEvent.id).join(
+        Event, Event.id == UserEvent.event_id
+        ).filter(
+        UserEvent.user_id == user_id, Event.start > datetime.now()
+        ).all()
```

Comment:
Hard to read

Reply:
the flake8 linter made me split lines way too many times. will documentation help?